kbase.yml

```diff
@@ -8,7 +8,7 @@ service-language:
     python
 
 module-version:
-    0.0.5
+    0.0.6
 
 owners:
     [jjeffryes]
```

Comment:
Please add yourself and dylan as addition owners. jjeffreyes left KBase quite some time ago.

Reply:
Added